content/en/docs/collector/management.md

```diff
@@ -81,129 +81,210 @@ Let's have a look at a concrete setup:
+Unblock-File -Path "opampsupervisor.exe"
+```
+
+{{% /tab %}} {{< /tabpane >}}
+
+#### Step 4 - Create an OpAMP Supervisor configuration file
+
+Create a file named `supervisor.yaml` with the following content:
 
 ```yaml
 server:
-  endpoint: ws://127.0.0.1:4320/v1/opamp
+  endpoint: wss://127.0.0.1:4320/v1/opamp
+  tls:
+    insecure_skip_verify: true
+
+capabilities:
+  accepts_remote_config: true
+  reports_effective_config: true
+  reports_own_metrics: false
+  reports_own_logs: true
+  reports_own_traces: false
+  reports_health: true
+  reports_remote_config: true
 
 agent:
   executable: $OTEL_COLLECTOR_BINARY
+
+storage:
+  directory: ./storage
 ```
 
-> **Note** Make sure to replace `$OTEL_COLLECTOR_BINARY` with the actual file
-> path. For example, in Linux or macOS, if you installed the collector in
-> `/usr/local/bin/` then you would replace `$OTEL_COLLECTOR_BINARY` with
-> `/usr/local/bin/otelcol`.
+{{% alert color="primary" title="NOTE" %}}
+
+Make sure to replace `$OTEL_COLLECTOR_BINARY` with the actual file path. For
+example, in Linux or macOS, if you installed the Collector in `/usr/local/bin/`,
+then you would replace `$OTEL_COLLECTOR_BINARY` with `/usr/local/bin/otelcol`.
+
+{{% /alert %}}
+
+#### Step 5 - Run the OpAMP Supervisor
 
-Next, create a collector configuration as follows (save it in a file called
-`effective.yaml` in the `./opamp-go/internal/examples/supervisor` directory):
+Now it's time to launch the supervisor, which in turn launches your
+OpenTelemetry Collector:
+
+```console
+$ ./opampsupervisor --config=./supervisor.yaml
+{"level":"info","ts":1745154644.746028,"logger":"supervisor","caller":"supervisor/supervisor.go:340","msg":"Supervisor starting","id":"01965352-9958-72da-905c-e40329c32c64"}
+{"level":"info","ts":1745154644.74608,"logger":"supervisor","caller":"supervisor/supervisor.go:1086","msg":"No last received remote config found"}
+```
+
+If everything worked out, you should now be able to go to
+[http://localhost:4321/](http://localhost:4321/) and access the OpAMP server UI.
+You should see your Collector listed among the agents managed by the Supervisor:
+
+![OpAMP example setup](../img/opamp-server-ui.png)
+
+#### Step 6 - Configure the OpenTelemetry Collector remotely
+
+Click on the Collector in the server UI and paste the following content to the
+`Additional Configuration` box:
 
 ```yaml
 receivers:
-  prometheus/own_metrics:
-    config:
-      scrape_configs:
-        - job_name: otel-collector
-          scrape_interval: 10s
-          static_configs:
-            - targets: [0.0.0.0:8888]
   hostmetrics:
     collection_interval: 10s
     scrapers:
-      load:
-      filesystem:
-      memory:
-      network:
+      cpu:
 
 exporters:
-  # NOTE: Prior to v0.86.0 use `logging` instead of `debug`.
   debug:
-    verbosity: detailed
```

Comment:
Why is this line removed? The detailed level is typically used with the debug exporter to show the full details of the data passed to it.

Reply:
Good catch, thanks!